File: TUGAS 2/tugas 2.1 .pas
{Nama    : Fauzy Ramadan}
{NPM     : 20.14.1.0051}
{Program : TGS 2.1 .pas}

program TGS2_1.pas;
uses crt;

var
p, l, luas : integer;

begin
writeln ('PROGRAM LUAS PERSEGI PANJANG');
writeln ('============================');
write ('Masukan Nilai Panjang = '); readln (p);
write ('Masukan Nilai Lebar = '); readln (l);
writeln;
luas := p*l;
writeln ('Luas = Panjang x Lebar');
writeln ('Luas = ', luas);
readln;

end.

File: TUGAS 2/TUGAS 2.1.cpp
// Nama : Fauzy Ramadan
// NPM : 20.14.1.0051

#include <iostream>
using namespace std;

int p, l, luas;
int main ()
{
	cout << "Program Luas Persegi Panjang"<<endl;
	cout << "============================"<<endl;
	cout << "Masukan nilai panjang : ";
	cin >> p;
	cout << "Masukan nilai lebar : ";
	cin >> l;
	
	cout << endl;
	luas = p*l;
	cout << "Luas = Panjang x Lebar"<<endl;
	cout << "Luas = "<< luas;
	
	return 0;
}
